corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -10,21 +10,21 @@
 
   <script type="text/html" id="property_template" class="hide">
     <tr>
-      <td class="col-sm-3">
+      <td class="col-sm-4">
         <input class="form-control" type="text" data-bind="value: property.name"/>
       </td>
-      <td class="col-sm-4">
+      <td class="col-sm-6">
         <input class="form-control" type="text" data-bind="value: property.label"/>
       </td>
       {% if js_options.search_prompt_appearance_enabled or js_options.default_value_expression_enabled%}
-      <td class="col-sm-5">
+      <td class="col-sm-4">
```

Comment:
Something of a nitpick here and below: bootstrap expects the number of columns to add to 12, so this should be 2.

Reply:
Sure